windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -149,7 +150,13 @@ func TestInstall(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	t.Cleanup(cancel)
 
-	fileServerAddr := mockRootfsFileServer(t, ctx)
+	fileServerAddr, closer := mockRootfsFileServer(t, ctx)
+	t.Cleanup(func() {
+		err := closer.Close()
+		if err != nil {
```

Comment:
nitpick: you can typically have a one liner here:

`if err := closer.Close(); err != nil` (somewhat a standard pattern when you return only one value, which is an error).